### ISSUE TITLE
Server crashes on attempting to fill an Empty Blaze Burner with Slugma

### ISSUE THREAD

**Issue by Vaerys-Dawn** [R3charged/Create-Slugma#8]
**Describe the bug**
Server crashes on attempting to fill an Empty Blaze Burner with Slugma

**To Reproduce**
1. Load Create Slugma on a server
2. Catch a Slugma
3. Place down an Empty Blaze Burner
4. Throw Slugma into the Burner

**Expected behavior**
Server doesn't crash on throw slugma

**Screenshots**
n/a

**Crash Report**
```
13.08 22:28:30 [Server] [INFO] java.lang.NoClassDefFoundError: net/minecraft/class_310
13.08 22:28:30 [Server] [INFO] at com.r3charged.common.createslugma.CobblemonUtils$Companion.spawnSendOutParticles(CobblemonUtils.java:31) ~[createslugma-1.20.1-fabric-1.0.1.jar:?]
13.08 22:28:30 [Server] [INFO] at com.cobblemon.mod.common.net.serverhandling.storage.SendOutPokemonHandler.handler$boa000$createslugma$onHandle(SendOutPokemonHandler.java:556) ~[Cobblemon-fabric-1.5.2+1.20.1.jar:?]
13.08 22:28:30 [Server] [INFO] at com.cobblemon.mod.common.net.serverhandling.storage.SendOutPokemonHandler.handle(SendOutPokemonHandler.java) ~[Cobblemon-fabric-1.5.2+1.20.1.jar:?]
13.08 22:28:30 [Server] [INFO] at com.cobblemon.mod.common.net.serverhandling.storage.SendOutPokemonHandler.handle(SendOutPokemonHandler.java:21) ~[Cobblemon-fabric-1.5.2+1.20.1.jar:?]
13.08 22:28:30 [Server] [INFO] at com.cobblemon.mod.common.api.net.ServerNetworkPacketHandler$DefaultImpls.handleOnNettyThread$lambda$0(ServerNetworkPacketHandler.java:19) ~[Cobblemon-fabric-1.5.2+1.20.1.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.class_3738.run(class_3738.java:18) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.class_1255.method_18859(class_1255.java:156) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.class_4093.method_18859(class_4093.java:23) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.server.MinecraftServer.method_24306(MinecraftServer.java:782) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.server.MinecraftServer.method_18859(MinecraftServer.java:164) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.class_1255.method_16075(class_1255.java:130) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.server.MinecraftServer.method_20415(MinecraftServer.java:764) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.server.MinecraftServer.method_16075(MinecraftServer.java:758) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.class_1255.method_5383(class_1255.java:115) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.server.MinecraftServer.method_16208(MinecraftServer.java:742) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:675) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265) ~[server-intermediary.jar:?]
13.08 22:28:30 [Server] [INFO] at java.lang.Thread.run(Thread.java:833) ~[?:?]
13.08 22:28:30 [Server] [INFO] Caused by: java.lang.ClassNotFoundException: net.minecraft.class_310
13.08 22:28:30 [Server] [INFO] at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
13.08 22:28:30 [Server] [INFO] at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
13.08 22:28:30 [Server] [INFO] at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226) ~[fabric-loader-0.16.0.jar:?]
13.08 22:28:30 [Server] [INFO] at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.16.0.jar:?]
13.08 22:28:30 [Server] [INFO] at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
13.08 22:28:30 [Server] [INFO] ... 18 more
```

**Comment by Vaerys-Dawn** [comment]
can confirm that this does not happen on singleplayer
![javaw_2024-08-14_08-36-28](https://github.com/user-attachments/assets/259b1e2d-3810-4d15-b65d-67c7f3386f0d)

**Comment by Vaerys-Dawn** [comment]
![Modrinth_App_2024-08-14_08-49-18](https://github.com/user-attachments/assets/42cb7bdf-d64f-4ce0-8f55-4cf3bc1aba51)

reproduction steps

**Comment by Vaerys-Dawn** [comment]
[crash-2024-08-14_08.49.15-server.txt](https://github.com/user-attachments/files/16606042/crash-2024-08-14_08.49.15-server.txt)

**Comment by Vaerys-Dawn** [comment]
https://hastebin.com/share/ixepenaniy.yaml full crash log

**Comment by R3charged** [comment]
Thanks for the report, fixed with Release 1.0.2
